app/workers/rapid_ready_for_decision/disability_compensation_job.rb

```diff
@@ -21,12 +21,11 @@ def perform(form526_submission_id)
 
       begin
         with_tracking(self.class.name, form526_submission.saved_claim_id, form526_submission_id) do
-          # Prefer "next" keyword here so that after-yield code continues execution
-          next if RapidReadyForDecision::Form526BaseJob.pending_eps?(form526_submission.auth_headers)
+          return if RapidReadyForDecision::Form526BaseJob.pending_eps?(form526_submission)
```

Comment:
Reverted this to `return` (and removed the comment) now that #9450 is merged, fixing a bug where `return` would skip some important clean-up logic.

Reply:
Great